compiler/src/iree/compiler/Codegen/LLVMCPU/Passes.cpp

```diff
@@ -131,6 +131,7 @@ static void addBufferizePasses(OpPassManager &passManager) {
   BufferizationOptions::MemCpyFn memcpyFn = cpuCopyFn;
   addIREEComprehensiveBufferizePasses(passManager, allocationFn, deallocationFn,
                                       memcpyFn);
+  passManager.addPass(memref::createFoldMemRefAliasOpsPass());
```

Comment:
I dont think you should do this here. You should add this somewhere https://github.com/openxla/iree/blob/3926ab7cb0edd9b3ee9d0b0d7c937412aec3816f/compiler/src/iree/compiler/Codegen/LLVMCPU/Passes.cpp#L741 (just before you call the emulation pass)

Reply:
I thought we want the cleanup after bufferization? Those subviews are tiling artifacts, and we can remove them after bufferization. I haven't looked into why there are regressions yet..

I tried moving it before emulation pass, but there is a bug in upstream patterns. It causes a crash in some pack codegen. That's a separate issue.